src/java.base/share/classes/java/nio/Direct-X-Buffer.java.template

```diff
@@ -72,14 +74,18 @@ class Direct$Type$Buffer$RW$$BO$
     // Cached unaligned-access capability
     static final boolean UNALIGNED = Bits.unaligned();
 
-    private record Deallocator(long address, long size, int capacity) implements Runnable {
+    private @Stable boolean temporary; // defaults to false
```

Comment:
Any reason this is `@Stable` instead of `final`?

Reply:
I think it avoids setting it to `false` when it is not `true`.